Bullet PR: Unable to build for Android

**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Manjaro Linux 17

**Issue description:**
<!-- What happened, and what was expected. -->
Godot with bullet physics won't build for Android on Linux

**Steps to reproduce:**
`git clone https://github.com/godotengine/godot`
`cd godot`

Apply the patch:
`wget https://patch-diff.githubusercontent.com/raw/godotengine/godot/pull/10013.patch`
`git am 10013.patch`

Build:
`scons platform=android target=release_debug`

**Logs:**
platform=android target=release_debug
scons: Reading SConscript files ...
Building for Android (armv7) (with NEON)
Checking for C header file mntent.h... (cached) yes
scons: done reading SConscript files.
scons: Building targets ...
[Initial build] Compiling shared ==> platform/android/os_android.cpp
[Initial build] Compiling shared ==> platform/android/godot_android.cpp
[Initial build] Compiling shared ==> platform/android/file_access_android.cpp
[Initial build] Compiling shared ==> platform/android/dir_access_android.cpp
[Initial build] Compiling shared ==> platform/android/audio_driver_opensl.cpp
[Initial build] Compiling shared ==> platform/android/file_access_jandroid.cpp
[Initial build] Compiling shared ==> platform/android/dir_access_jandroid.cpp
[Initial build] Compiling shared ==> platform/android/thread_jandroid.cpp
[Initial build] Compiling shared ==> platform/android/audio_driver_jandroid.cpp
[Initial build] Compiling shared ==> platform/android/ifaddrs_android.cpp
[Initial build] Compiling shared ==> platform/android/android_native_app_glue.c
[Initial build] Compiling shared ==> platform/android/java_glue.cpp
[Initial build] Compiling shared ==> platform/android/cpu-features.c
[Initial build] Compiling shared ==> platform/android/java_class_wrapper.cpp
[Initial build] Compiling ==> main/performance.cpp
[Initial build] Compiling ==> main/input_default.cpp
[Initial build] make_app_icon(["main/app_icon.gen.h"], ["main/app_icon.png"])
[Initial build] make_splash(["main/splash.gen.h"], ["main/splash.png"])
[Initial build] Compiling ==> main/main.cpp
[Initial build] Linking Static Library ==> main/libmain.android.opt.debug.armv7.neon.a
Ranlib Library         ==> main/libmain.android.opt.debug.armv7.neon.a
[Initial build] Compiling ==> main/tests/test_physics.cpp
[Initial build] Compiling ==> main/tests/test_render.cpp
[Initial build] Compiling ==> main/tests/test_physics_2d.cpp
[Initial build] Compiling ==> main/tests/test_gdscript.cpp
[Initial build] Compiling ==> main/tests/test_shader_lang.cpp
[Initial build] Compiling ==> main/tests/test_gui.cpp
[Initial build] Compiling ==> main/tests/test_ordered_hash_map.cpp
[Initial build] Compiling ==> main/tests/test_io.cpp
[Initial build] Compiling ==> main/tests/test_main.cpp
[Initial build] Compiling ==> main/tests/test_oa_hash_map.cpp
[Initial build] Compiling ==> main/tests/test_string.cpp
[Initial build] Compiling ==> main/tests/test_image.cpp
[Initial build] Compiling ==> main/tests/test_math.cpp
[Initial build] Linking Static Library ==> main/tests/libtests.android.opt.debug.armv7.neon.a
Ranlib Library         ==> main/tests/libtests.android.opt.debug.armv7.neon.a
[Initial build] Compiling ==> modules/register_module_types.gen.cpp
[Initial build] Compiling ==> thirdparty/Bullet/src/BulletCollision/BroadphaseCollision/btAxisSweep3.cpp
In file included from thirdparty/Bullet/src/BulletCollision/BroadphaseCollision/btAxisSweep3.cpp:20:
In file included from thirdparty/Bullet/src/BulletCollision/BroadphaseCollision/btAxisSweep3.h:23:
In file included from thirdparty/Bullet/src/BulletCollision/BroadphaseCollision/btOverlappingPairCache.h:24:
thirdparty/Bullet/src/LinearMath/btAlignedObjectArray.h:39:10: fatal error: 
      'new' file not found
#include <new> //for placement new
         ^~~~~
1 error generated.
scons: *** [thirdparty/Bullet/src/BulletCollision/BroadphaseCollision/btAxisSweep3.android.opt.debug.armv7.neon.o] Error 1
scons: building terminated because of errors.
`

--- a/platform/android/detect.py
+++ b/platform/android/detect.py
@@ -25,7 +25,7 @@ def get_opts():
         ('ndk_platform', 'Target platform (android-<api>, e.g. "android-18")', "android-18"),
         EnumVariable('android_arch', 'Target architecture', "armv7", ('armv7', 'armv6', 'arm64v8', 'x86')),
         BoolVariable('android_neon', 'Enable NEON support (armv7 only)', True),
-        BoolVariable('android_stl', 'Enable Android STL support (for modules)', False),
+        BoolVariable('android_stl', 'Enable Android STL support (for modules)', True),
         BoolVariable('ndk_unified_headers', 'Enable NDK unified headers', True)
     ]
 

